playbooks/roles/ocp-network-observability/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# defaults file for ocp-network-observability
+noo_role_enable: false
+noo_deploy: false
+noo_directory: "/tmp/network-observability"
+noo_catalogsource_image: "brew.registry.redhat.io/rh-osbs/iib:<iib-image-index>"
+noo_installation_source: "Internal"
+noo_catalogsource_name: "qe-app-reigstry" 
+noo_loki_operator: "Released"
+noo_lokistack_size: "1x.extra-small"
+noo_e2e_repo_branch: "release-<ocp-version>"
```

Comment:
This will fail as a default. Set this var as required or set a specific ocp version.

Reply:
Will set the default values as `master` for repo branch. User can set the release branch for specific OCP version as mentioned in the role README page. The operator image var for custom catalogsource `noo_catalogsource_image` may need to set as required input for user to avoid execution fail.